cmd/main.go

```diff
@@ -181,10 +186,33 @@ func (f *Aggregator) Aggregate(targets []string, output io.Writer) {
 					}
 					if existingMf, ok := allFamilies[mfName]; ok {
 						for _, m := range mf.Metric {
-							existingMf.Metric = append(existingMf.Metric, m)
+							for _, r := range m.Label {
+								match, _ := regexp.MatchString(*targetLabelRegexp, r.GetValue())
+								if match {
+									existingMf.Metric = append(existingMf.Metric, m)
```

Comment:
won't this duplicate the metric as many times as there are matching labels?

Reply:
Hmm, yes I think you are right, now I see something strange, let me try to fix it.

Reply:
Should be fixed now, added some break's